jest.config.js

```diff
@@ -1,22 +1,27 @@
-const { pathsToModuleNameMapper } = require("ts-jest");
-const { compilerOptions } = require("../tsconfig.json");
+require("dotenv").config({ path: ".env.test" })
 
-const paths = compilerOptions.paths ? compilerOptions.paths : {};
+const { pathsToModuleNameMapper } = require("ts-jest")
+const { compilerOptions } = require("./tsconfig.json")
+
+const paths = compilerOptions.paths ? compilerOptions.paths : {}
 
 module.exports = {
-  rootDir: "../",
+  rootDir: "./",
   testEnvironment: "jsdom",
   setupFilesAfterEnv: ["<rootDir>/test/setup.js"],
   testPathIgnorePatterns: ["<rootDir>/node_modules/"],
   transform: {
-    "^.+\\.(js|ts|tsx)$": "ts-jest"
+    "^.+\\.(js|ts|tsx)$": [
+      "@swc/jest",
+      { jsc: { transform: { react: { runtime: "automatic" } } } },
```

Comment:
Use `@swc/jest` instead of `ts-jest` for faster test. This means our test files are not type checked but it's an acceptable trade off since the editor (eg. vscode) should catch any type errors.